Helpers/ConfigHelper.php

```diff
@@ -172,4 +192,38 @@ protected function getConfigPath($configCode): string
         return self::XML_PATH_PAYMENT . '/' . self::XML_PATH_METHODE . '/' . $configCode;
     }
 
+    /**
+     * @param array $plans
+     *
+     * @return void
+     */
+    public function saveBasePlansConfig(array $plans): void
+    {
+        $this->saveConfig(self::BASE_PLANS_CONFIG, $this->serializer->serialize($plans), $this->storeHelper->getScope(), $this->storeHelper->getStoreId());
+        $this->saveConfig(self::BASE_PLANS_TIME, time(), $this->storeHelper->getScope(), $this->storeHelper->getStoreId());
```

Comment:
Could you explain me why you save the time here ?

Reply:
Feature not implemented

Reply:
remove